release-notes/topics/hub-460.adoc

```diff
@@ -0,0 +1,30 @@
+[[hub-460-intro]]
+= Automation Hub 4.6
+
+Automation Hub allows you to discover and utilize new certified automation content from Red Hat Ansible and Certified Partners. On Ansible Automation Hub, you can discover and manage Ansible Collections, which is supported automation content developed by both partners and Red Hat for use cases such as cloud automation, network automation, security automation, and more.
+
+.Enhancements
+
+* Adopted the new pulp RBAC system.
+* Added a configurable automatic logout time. Set a minimum password length for internal users. 
+* Added the capability to configure LDAP with private automation hub.
+* Added visibility for execution environments created by ansible-builder in the automation hub UI.
+* Fixed an error when navigating to a non-existent group URL.
+* Fixed an issue where roles could not be created through the UI.
+* Fixed an issue during Hub installation during the collect static content task
```

Comment:
Suggest changing to: Fixed an issue with Hub installation during the collect static content task.

Reply:
Fixed.